docker/README.md

```diff
@@ -0,0 +1,28 @@
+### Docker Usage Instructions
+
+1. Build docker image
+
+   ```
+   docker build -t e-mission-server:latest .
+   ```
+
+2. Create docker network
+
+   ```
+   docker network create e-mission --driver=bridge
+   ```
+   
+3. Run the server
+
+   Assuming `config` files are located at `docker/config` folder
+
+   ```
+   docker run -d \
+     -p 8080:8080 \
+     --name=e-mission-server-1 \
+     --net="e-mission" \
+     --mount type=bind,source="$(pwd)"/docker/config/db.conf,target=/usr/src/app/conf/storage/db.conf,readonly \
```

Comment:
This is pretty cool. I assume I leave out this and the next two lines if I just want to start the container with defaults?
Because for development, people may want to just start the app and not want to figure out how to set config files.

Reply:
You probably want to have development versus production/deployment scripts anyway because the port mapping will be different. Development maps `8080:8080` but I would assume production should do `443:443`. And production will want to configure SSL certificates as well.

Reply:
For production, we are using another script that runs as a "web gateway". This helps us separate the problem and handle the SSL modularly. Again, I will be submitting the files in `e-mission-docs` repo along with the mongodb.

Reply:
@atton16 I think that the production script should go here as well. Basically, I think that there should be two separate scripts, one for each conceptual module. After the UI component is separated from the server component (#596), we could have three.

We want to have as few script locations as possible - fewer commands to type out - while still supporting flexible deployments. So basically one script per _module_ or _tier_